tests/defences/preprocessor/test_spatial_smoothing_tensorflow.py

```diff
@@ -120,7 +120,3 @@ def test_relation_clip_values_error(self, is_tf_version_2):
             exc_msg = "Invalid 'clip_values': min >= max."
             with pytest.raises(ValueError, match=exc_msg):
                 SpatialSmoothingTensorFlowV2(clip_values=(1, 0))
-
```

Comment:
Should similar changes as in file `test_spatial_smoothing_pytorch.py` apply to this file?

Reply:
yes indeed. Thanks for pointing it out, I must have skipped that file